src/components/Sidebar/SidebarOption.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import './SidebarOption.css';
+import PropTypes from 'prop-types';
+
+// eslint-disable-next-line react/prop-types
```

Comment:
Kindly remove this line so that linters rules are followed and the code is written with best practices. 

Just an FYI, 💼 react/prop-types rule is enabled in the recommended [config](https://github.com/jsx-eslint/eslint-plugin-react/#shareable-configs). Defining types for component props improves reusability of your components by validating received data. It can warn other developers if they make a mistake while reusing the component with improper data type. [Sourcing from this link](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/prop-types.md)

**Additionally, you can resolve this error by fixing the suggestion given on `line 15`**